tools/scoring/score/dimensions/entity_connection_identification.py

```diff
@@ -25,6 +25,9 @@
 class EntityConnectionIdentification(Dimension):
   """Quantifies whether connections between entities were
   correctly and completely defined in the proposed file."""
+
+  category = DimensionCategories.COMPLEX
```

Comment:
how do you determine if it is a simple or complex one for a connection?

Reply:
Connections are a feature of both reporting and virtual entities, so to score them we need to pass the deserialized files rather than the pre-matched list of translations. Simple dimensions receive translations whereas complex dimensions receive deserialized files.

Reply:
cool can you update the comment for the definitions of Simple vs complex with your comment above ? :)